indico/modules/categories/client/js/management.js

```diff
@@ -308,6 +279,47 @@ import {$T} from 'indico/utils/i18n';
         },
       },
     }).isEverythingSelected;
+
+    _fetchSourceCategory(categoryId);
+
+    [].forEach.call(
```

Comment:
Is (and line 306) this for browser compatibility? Is it worth it? (Not sure which browser versions started supporting NodeList forEach).

Reply:
oh nice, I didn't know they added forEach to NodeList. We didn't really bump the browser version requirement with the 3.0 release so I think for now I'll stay on the safe side and keep it..

https://caniuse.com/mdn-api_nodelist_foreach